pkgs/applications/office/gnucash/default.nix

```diff
@@ -97,12 +98,29 @@ stdenv.mkDerivation rec {
   enableParallelChecking = true;
   checkTarget = "check";
 
+  passthru.docs = stdenv.mkDerivation {
+    pname = "gnucash-docs";
+    inherit (finalAttrs) version;
+
+    src = fetchFromGitHub {
+      owner = "Gnucash";
+      repo = "gnucash-docs";
+      rev = finalAttrs.version;
+      hash = "sha256-aPxQEcpo8SPv8lPQbxMl1wg8ijH9Rz0oo4K5lp3C/bw=";
+    };
+
+    nativeBuildInputs = [ cmake ];
+    buildInputs = [ libxml2 libxslt ];
+  };
+
   preFixup = ''
     gappsWrapperArgs+=(
+      # documentation
+      --prefix XDG_DATA_DIRS : ${finalAttrs.passthru.docs}/share
```

Comment:
This should probably suffix?

Reply:
I think preffix is correct. It results in env as follows for me:

```console
$ strings /proc/$(pgrep gnucash)/environ | grep XDG_DATA_DIRS | tr : \\n
XDG_DATA_DIRS=/nix/store/csr1hndyx8p0mi61z6pp2ppi3fqd3sg2-gnucash-docs-5.4/share
/nix/store/w0n0lc2n5zdkrgw5glmzhygyfwcaqgxf-gnucash-5.4/share
/nix/store/n04wvmncnbg7griy4myhy1k19bj9m0yy-gsettings-desktop-schemas-44.0/share/gsettings-schemas/gsettings-desktop-schemas-44.0
/nix/store/5y5gmfp3pjr60rlsvrc1zk8pds14kzzv-gtk+3-3.24.38/share/gsettings-schemas/gtk+3-3.24.38
/nix/store/w0n0lc2n5zdkrgw5glmzhygyfwcaqgxf-gnucash-5.4/share/gsettings-schemas/gnucash-5.4
/nix/store/hq05zfryh5ck0ay38wna8aa0bp1c3bh8-desktops/share
/home/nevivurn/.nix-profile/share
/etc/profiles/per-user/nevivurn/share
/nix/var/nix/profiles/default/share
/run/current-system/sw/share
```

Which feels like it puts `gnucash-docs` in the correct position.